SmartDeviceLinkTests/DevAPISpecs/SDLListFilesOperationSpec.m

```diff
@@ -92,13 +92,24 @@
                 badResponse.success = @NO;
                 badResponse.spaceAvailable = responseSpaceAvailable;
                 badResponse.filenames = responseFileNames;
-                
-                [testConnectionManager respondToLastRequestWithResponse:badResponse error:[NSError sdl_lifecycle_unknownRemoteErrorWithDescription:responseErrorDescription andReason:responseErrorReason]];
             });
             
             it(@"should have called completion handler with error", ^{
+                [testConnectionManager respondToLastRequestWithResponse:badResponse error:[NSError sdl_lifecycle_unknownRemoteErrorWithDescription:responseErrorDescription andReason:responseErrorReason]];
+
+                expect(errorResult.localizedDescription).to(match(responseErrorDescription));
+                expect(errorResult.localizedFailureReason).to(match(responseErrorReason));
+                expect(@(successResult)).to(equal(@NO));
+            });
+
+            it(@"should have called completion handler with error including a resultCode ", ^{
```

Comment:
Based on the above change request, I think that this `it` is the only one needed?